CLAUDE.md

```
@@ -36,15 +36,47 @@ docker run --env-file .env -p 8051:8051 mcp/crawl4ai-rag
 python test_neo4j_setup.py
 ```
 
-### Testing
+### Testing and Quality Assurance
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)